sdk/resourcemanager/Azure.ResourceManager/src/ArmResource.cs

```diff
@@ -97,12 +98,6 @@ internal ArmResource(ClientContext clientContext, ResourceIdentifier id)
         /// </summary>
         protected internal virtual HttpPipeline Pipeline { get; }
 
-        /// <summary>
```

Comment:
This class should remove ValidateResourceType method and the reference to it in the ctor